src/get_session_access_token_test.ts

```diff
@@ -32,27 +49,42 @@ Deno.test("getSessionAccessToken()", async (test) => {
     );
   });
 
-  await test.step("rejects for an expired access token", async () => {
+  await test.step("rejects for an access token which expires in less than 5 seconds", async () => {
     const sessionId = crypto.randomUUID();
     const tokens: Tokens = {
       ...genTokens(),
-      expiresIn: 0,
+      refreshToken: crypto.randomUUID(),
+      expiresIn: 3,
     };
     await setTokens(sessionId, tokens);
-    assertRejects(async () =>
+    await assertRejects(async () =>
```

Comment:
I also found that the `assertRejects` calls were not awaited and were causing false-positives in test as well